### ISSUE TITLE
error on predict

### ISSUE THREAD

**Issue by rsteca** [salestock/fastText.py#68]
Doing the following:
```
import fasttext
classifier = fasttext.supervised('data.train.txt', 'model')
classifier.predict(['some text'])
```

gives me the following error:
```
Segmentation fault (core dumped)
```

But if I use the fastText library directly from shell it works ok. 
I have tried multiple datasets and it always gives the same error.

**Comment by pyk** [comment]
Hi @rsteca , thanks for the report!

For now please use this:

```python
# Load your previously trained model
classifier = fasttext.load_model('model.bin')
classifier.predict(['some text'])
```

Will try to investigate this problem

**Comment by pyk** [comment]
Solved by #69 , Will release `v0.7.6` with the patch
